File: hephasteus-frontend/src/app/model/Clothing.ts
import {ClothesCatergory} from "./ClothesCatergory";
import {ClothesBrand} from "./ClothesBrand";

export class Clothing {
  id!:string;
  name!:string;
  description!:string;
  image!:string;
  colors:string[]=[];
  clothesCategory!:ClothesCatergory;
  clothesBrand!:ClothesBrand;
}

File: fmi-back-end-main/src/main/java/com/fmi/demo/domain/model/Clothing.java
package com.fmi.demo.domain.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;


@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Clothing  {

    protected String id;
    private String name;
    private String description;
    private String image;
    private List<String> colors;
    ClothesCategory clothesCategory;
    ClothesBrand clothesBrand;

}
